tests/bdd/features/snapshot/delete/delete.feature

```diff
@@ -0,0 +1,73 @@
+Feature: Volume Snapshot deletion
+
+  Background:
+    Given a deployer cluster
+
+  # USUAL DELETE OPERATION
+  Scenario Outline: Snapshot deletion volume
+    Given a <publish_status> volume with single replica
+    And a snapshot is created for the volume
+    When the created snapshot is deleted
+    Then the snapshot deletion should not fail
+    And the snapshot should not be present upon listing
+    Examples:
+      | publish_status |
+      | published      |
+      | unpublished    |
+
+
+  # USUAL DELETE OPERATION WITHOUT SOURCE
+  Scenario Outline: Snapshot deletion volume after volume deletion
+    Given a <publish_status> volume with single replica
+    And a snapshot is created for the volume
+    And the source volume is deleted
+    When the created snapshot is deleted
```

Comment:
```suggestion
    When the source volume is deleted
    And the created snapshot is deleted
```

hmm or perhaps make the first 3 a background, since they seem shared in quite a few?

Reply:
Yeah possibly.